x-pack/plugins/cases/public/components/description/index.test.tsx

```diff
@@ -45,48 +39,42 @@ describe.skip('Description', () => {
   it('renders description correctly', async () => {
     appMockRender.render(<Description {...defaultProps} onUpdateField={onUpdateField} />);
 
-    expect(screen.getByTestId('description')).toBeInTheDocument();
-    expect(screen.getByText('Security banana Issue')).toBeInTheDocument();
+    expect(await screen.findByTestId('description')).toBeInTheDocument();
+    expect(await screen.findByText('Security banana Issue')).toBeInTheDocument();
   });
 
   it('hides and shows the description correctly when collapse button clicked', async () => {
-    const res = appMockRender.render(
-      <Description {...defaultProps} onUpdateField={onUpdateField} />
-    );
+    appMockRender.render(<Description {...defaultProps} onUpdateField={onUpdateField} />);
 
-    userEvent.click(res.getByTestId('description-collapse-icon'));
+    userEvent.click(await screen.findByTestId('description-collapse-icon'));
 
     await waitFor(() => {
```

Comment:
Im not sure this `waitFor` is necessary.

Reply:
It is needed because in the beginning, the description is open, and then when we click the toggle we are waiting for the description to disappear. I tried with `waitForElementToBeRemoved` but it did not work.